src/domain/session/room/timeline/tiles/BaseTextTile.js

```diff
@@ -56,4 +57,43 @@ export class BaseTextTile extends BaseMessageTile {
         }
         return this._messageBody;
     }
+
+    get replyPreviewBody() {
+        if (!this._entry.contextEventId) {
+            return null;
+        }
+        const entry = this._entry.contextEntry;
+        const error = this._generateError(entry);
+        if (error?.name === "ContextEntryNotFound") {
+            return { error };
+        }
+        const format = entry.content.format === "org.matrix.custom.html" ? BodyFormat.Html : BodyFormat.Plain;
+        const body = entry.content["formatted_body"] ?? entry.content["body"];
```

Comment:
if format is plain, we should never look at formatted_body.
Perhaps we can extract two functions to get the format from an event and the body from an event given the format and use that here and in TextTile where it is already implemented?

Reply:
Marking as resolved since we're just using `TextTile` as the vm.